pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -18,6 +17,9 @@
 , isGNU ? false, isClang ? cc.isClang or false, gnugrep ? null
 , buildPackages ? {}
 , libcxx ? null
+, gccForLibs ? if useCcForLibs then cc else null
+# same as `gccForLibs`, but generalized beyond clang
+, useCcForLibs ? isClang || (cc.passthru.enableNixDrivenBootstrap or false)
```

Comment:
Worth leaving `cc.passthru.enableNixDrivenBootstrap` out for now in this PR? Otherwise looks OK.